components/pkg-export-kubernetes/src/manifest.rs

```diff
@@ -101,6 +101,8 @@ impl Manifest {
                           .map(str::to_string)
                           .unwrap_or_else(|| format!("{}-{}", pkg_ident.name, version_suffix));
 
+        let formatted_resource_name = Manifest::formatted_metadata_name(&name);
```

Comment:
Should this variable be called `formatted_metadata_name`, or should the function be called `Manifest::formatted_resource_name`?

(Just curious what, if any, the distinction is)

Reply:
There's not, I'm happy to go either way.

Reply:
I'd say we should use whichever one is more natural / appropriate in a Kubernetes context (if indeed one actually _is_ more appropriate than the other).

Then have the variable named similarly to the function(any time there's a difference, I always wonder if it's significant :smile: )

Reply:
Fixing this now!

Reply:
Fixed!